2016-11-10 12:56:34,401 - har_predict_aggr - DEBUG - initializing Processor
2016-11-10 12:56:34,627 - har_predict_aggr - INFO - ==> Check if data model folders exist in HDFS and get data model paths and valid user list
ls: Call From hpe-smartwatch-03.hpintelco.org/10.3.35.93 to localhost:9000 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused
2016-11-10 12:56:36,779 - har_predict_aggr - ERROR -     Data model home path(/tmp/models) DOES NOT exist.
==> Check if this process is running or not ...
==> Start rta_wrapper ...
==> Generate process ID file har_predict_aggr_ensemble_1_1_naivebayes.pid in path /opt/rta_hc/rta-hc-framework-master/run ...
==> Start running the process ...
